docs/developer-docs/multi-chain/chain-key-tokens/ckerc20/making-transactions.mdx

```diff
@@ -0,0 +1,120 @@
+
+:::warning
+
+If your encoded principal ID is incorrect, funds will be lost.
+
+:::
+
+No further user action is required. The icrc1_balance_of function can be called on the corresponding ckERC20 ledger to check if the ckERC20 tokens have been minted, which usually takes about 20 minutes.
+
+
+## Converting ckERC20 tokens to ERC-20 tokens
+
+Since the user must pay the transaction fees, the `icrc2_approve` function must be called on the ckETH ledger, allowing the ckETH minter to spend some of the user's ckETH tokens.
+For example, the following command approves the ckETH minter to take up to 1 ckETH from the user's account.
+
+```
+dfx canister --ic call ledger icrc2_approve "(record { spender = record { owner = principal \"$(dfx canister id minter --ic)\" }; amount = 1_000_000_000_000_000_000:nat })"
+```
+
+The user must further approve the ckETH minter to transfer the desired amount of the targeted ckERC20 token:
+
+```
+dfx canister --ic call CKERC20_LEDGER icrc2_approve "(record { spender = record { owner = principal \"$(dfx canister id minter --ic)\" }; amount = CKERC20_TOKEN_AMOUNT:nat })"
+```
+
+The placeholder `CKERC20_LEDGER` must be replaced by the canister ID of the corresponding ckERC20 ledger and `CKERC20_TOKEN_AMOUNT` must be replaced by the desired amount, denominated in the smallest unit of the token. For example, USDC and ckUSDC use 6 decimals, so 1 USDC must be written as `1_000_000`.
+
+The last step is to call the `withdraw_erc20` function on the ckETH minter as follows.
+
+```
+dfx canister --ic call minter withdraw_erc20 "(record { amount = CKERC20_TOKEN_AMOUNT:nat; ckerc20_ledger_id = principal \"CKERC20_LEDGER\"; recipient = \"ETH_ADDRESS\"})"
+```
+
+As before, `CKERC20_TOKEN_AMOUNT` and `CKERC20_LEDGER` must be replaced by the desired amount in the smallest unit and the canister ID of the ckERC20 ledger, respectively.
+Lastly, `ETH_ADDRESS` must be replaced by the desired Ethereum destination address.
+
+## Transferring ckERC20 tokens
+
+ckERC20 tokens are standard ICRC-1 tokens and can be transferred by calling the `ircr1_transfer` function.
+
+```
+dfx canister --ic call CKERC20_LEDGER "(record { to = DESTINATION_ACCOUNT; amount = AMOUNT:nat})"
+```
+
+The placeholders `CKERC20_LEDGER`, `DESTINATION_ACCOUNT`, and `AMOUNT` must be replaced by the canister ID of the targeted ckERC20 ledger, the destination account, and the desired amount, respectively. More information about the `icrc1_tranfer` function can be found in the [ICRC-1 standard specification](https://github.com/dfinity/ICRC-1/tree/main/standards/ICRC-1).
+
+:::note
+
+Transferring ckERC20 tokens incurs a small fee. The exact amount depends on the specific ckERC20 token. The transfer fee can be obtained by calling the `icrc1_metadata` function on the ckERC20 ledger. The response contains the fee under the key `icrc1:fee`.
+
+:::
+
+## Viewing ckERC20 token transactions
+
+The ckERC20 tokens have not been added to the Internet Computer dashboard yet.
```

Comment:
Aren't the ckUSDC transactions on the dashboard? https://dashboard.internetcomputer.org/ethereum/xevnm-gaaaa-aaaar-qafnq-cai

IMO we should include mention of ckUSDC here and the link to the transaction dashboard.

Reply:
You're right. Last time I checked, these transactions were not there yet.   :slightly_smiling_face: 
I'll add a paragraph mentioning concrete ckERC20 tokens including links here.

Reply:
Done! Please take another look.